README

```diff
@@ -40,8 +40,9 @@ GNU versions of these tools are recommended.
 
 Installation
 ------------
-Installing Geany is done by the following three commands:
-$ ./configure
+Installing Geany is done by the following commands:
+$ libtoolize
+$ ./autogen.sh
```

Comment:
It needs to be called any time `configure` script needs to be re-generated also, like if someone edits `configure.ac` or any of the `.m4` files. I think that's why `autogen.sh` calls `configure` for you, to avoid the extra step.